pkgs/by-name/ai/aiken/package.nix

```diff
@@ -10,16 +10,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "aiken";
-  version = "1.0.29-alpha"; # all releases are 'alpha'
+  version = "1.1.0"; # all releases are 'alpha'
```

Comment:
Closing this to re-open with comment removed as I cannot force push to this branch